mlir/include/mlir/Conversion/Passes.td

```diff
@@ -694,6 +694,34 @@ def ConvertParallelLoopToLuminousLaunch :
                            "async::AsyncDialect"];
 }
 
+//===----------------------------------------------------------------------===//
+// LuminousToStandard
+//===----------------------------------------------------------------------===//
+
+def ConvertLuminousToStandard :
+    Pass<"convert-luminous-to-std", "ModuleOp"> {
+  let summary = "Convert luminous module ops to module ops";
```

Comment:
It also converts luminous func, please add.  Also maybe add `description` and give an example with input and output